src/js/components/aboutTheData/componentMapping/tooltipContentMapping.jsx

```diff
@@ -101,5 +101,15 @@ export const columnTooltips = {
         <p>
             Agency Comments are optional and provided by agencies at the time they submit their data to USAspending.gov in the required dataset formats (File A, B, C, D1, and D2). For more information about the DATA Act reporting flow, visit <a target="_blank" rel="noopener noreferrer" href="https://fiscal.treasury.gov/files/data-transparency/daims-information-flow-diagram.pdf">https://fiscal.treasury.gov/files/data-transparency/daims-information-flow-diagram.pdf</a>
         </p>
+    ),
+    percentOfBudgetSubmissions: (
+        <p>
+            This is an agency&apos;s total budgetary resources for the fiscal year through the selected period as a portion of all agency budgetary resources to-date.
+        </p>
+    ),
+    percentOfBudgetPublications: (
```

Comment:
This "feels" like a weird change. Why not use the column name? Or switch them all to IDs (probably better)?

Reply:
This column alone requires different tooltip content in each tab for the same column name. The other columns' tooltip content matches between the two tabs, so they essentially use the display name as the id.